Add a note about best practice for choice of a backup store. (#780)
longhorn/website: content/docs/1.6.0/best-practices.md

## Volume Maintenance

> We highly recommend using the built-in backup feature of Longhorn.  A backup can be saved to an object store (S3, Azure, etc), to an NFS server, or to a SMB/CIFS server, although an object store is the preferred method.

Reviewer: ```suggestion
Using Longhorn's built-in backup feature is highly recommended. You can save backups to an object store (such as S3 and Azure), an NFS server, or an SMB or CIFS server. Saving to an object store is preferable. 
```

Reviewer: I edited the paragraph based on the doc style guidelines (for example, avoid using "we" and break up long sentences whenever possible).
